lib/cylc/config.py

```diff
@@ -1822,9 +1822,14 @@ def load_graph(self):
 
         # Parse and process each graph section.
         for section, graph in sections:
-            seq = get_sequence(section,
-                               self.cfg['scheduling']['initial cycle point'],
-                               self.cfg['scheduling']['final cycle point'])
+            try:
+                seq = get_sequence(
+                    section,
+                    self.cfg['scheduling']['initial cycle point'],
+                    self.cfg['scheduling']['final cycle point'])
+            except (AttributeError, TypeError):
+                raise SuiteConfigError(
+                    "ERROR: Invalid recurrence representation: %s" % section)
```

Comment:
Looking at the statement in the `try:` block above, the argument to the `get_sequence` function are the section title (which should be a cycle point recurrence), the initial cycle point and the final cycle point.

The initial and final cycle points should already have their syntaxes checked earlier, so the only thing that can cause exception here should be invalid or unsupported recurrence syntax, or a bug in our system.